enhancements/etcd/etcd-encryption-for-separate-oauth-apis.md

```diff
@@ -0,0 +1,149 @@
+---
+title: ETCD-Encryption-For-Separate-OAuth-APIs
+authors:
+  - "@p0lyn0mial"
+reviewers:
+  - "@sttts"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2020-03-17
+last-updated: 2020-03-17
+status: implementable
+see-also: https://github.com/openshift/enhancements/blob/master/enhancements/authentication/separate-oauth-resources.md, https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
+replaces:
+superseded-by:
+---
+
+# ETCD Encryption For Separate OAuth APIs
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+## Summary
+
+The `encryption-config` used by OpenShift API server to encrypt/decrypt resources will be also used by the new `oauth-apiserver`.
```

Comment:
... for one release (4.5) and will be split in the next (4.6), in order to allow seamless upgrade and downgrade of encrypted servers.